db/migrations/campaign/V4.0.0.1__table_campaigns.sql

```diff
@@ -1,48 +1,51 @@
 BEGIN TRANSACTION;
 
-CREATE TABLE IF NOT EXISTS "CampaignTypes" (
-    "Id" uuid NOT NULL DEFAULT (gen_random_uuid()),
-    "Name" character varying(50) NOT NULL,
-    "DescriptionKey" character varying(50) NULL,
-    CONSTRAINT "PK_CampaignTypes" PRIMARY KEY ("Id")
+CREATE TABLE IF NOT EXISTS campaign.campaign_types (
```

Comment:
@xenry I'm not sure if editing the same migration will have the expected/desired effect when delivered to production.

We might need to drop the tables and create them in a separate migration like `V4.0.0.2__table_campaigns.sql

Reply:
yep, good idea, will fix it later today

Reply:
@kachar fixed :)